src/main/java/com/example/networkautomation/routes/PathsConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.example.networkautomation.routes;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+
+/**
+ * This is a Javadoc comment. It provides information about the method or class.
+ * You can include a description, parameter explanations, return value, exceptions, etc.
+ */
+@Configuration
+@EnableWebMvc
+public class PathsConfig implements WebMvcConfigurer {
+    /**
+     * This is a Javadoc comment. It provides information about the method or class.
+     * You can include a description, parameter explanations, return value, exceptions, etc.
+     */
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addViewController("/").setViewName("home");
+        registry.addViewController("/home").setViewName("home");
```

Comment:
How does this `addViewControllers` works? and where did you declared the home view and how are you rendering the html page? explain me in details

Reply:
This is a method under WebMvcConfigurer class by using this method I am configuring a specific URL to a specific view. Here, addViewController mapped the "/" and "/home" URL's endpoints to the view "home". The views are put into the resourses/templates folders from there our application showing the view.